invenio_github/views/github.py

```diff
@@ -67,7 +67,8 @@ def create_ui_blueprint(app):
         url_prefix="/account/settings/github",
     )
     if app.config.get("GITHUB_INTEGRATION_ENABLED", False):
-        register_ui_routes(blueprint)
+        with app.app_context():
```

Comment:
Can you add please here the comment about removing this when the proper fix will be merged?